doc/versions.json

```diff
@@ -20,12 +21,21 @@
   ],
   "COMPONENTS_BY_VERSION": {
     "latest": {
-      "ncs": "1.6.99",
-      "nrfxlib": "1.6.99",
+      "ncs": "1.7.0",
+      "nrfxlib": "1.7.0",
       "zephyr": "2.6.99",
       "mcuboot": "1.7.99",
       "nrfx": "2.5",
-      "tfm": "1.3.99",
+      "tfm": "1.3.0",
```

Comment:
@gmarull, to make the tfm versions identical, in addition to this change, should we make changes to tfm/conf.py also (like set _version = "1.7.0"_)

Reply:
do you guys want to make the changes in this PR?

Reply:
I would prefer a separate PR.

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/5616